homework-submission/week1-react/src/component/update.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+class Update extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            description: props.text,
+            deadline: props.date
+        }
+        this.handleDescription = this.handleDescription.bind(this);
+        this.handleDeadline = this.handleDeadline.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    
+    handleDescription (event) {
+      this.setState({
+          description: event.target.value,
+      })
+    }
+
+    handleDeadline (event) {
+        this.setState({
+            deadline: event.target.value
+        })
+    }
+
+    handleSubmit (event) {
```

Comment:
Please look this up: https://github.com/HackYourFuture/React/pull/274#discussion_r235436549

Reply:
To understand what you advise, I took a look at stackoverflow. I see this:
"When you use the <button> element without specifying a type, it defaults to type="submit". You need to specify it as type="button" to stop it submitting the form:"
And then I removed `handleSubmit` function and changed `type="submit"` as `type="button"`  Now it works as previously. But I could not realize the difference